### ISSUE TITLE
Feature Request: TLS certificate creation/enrollment using MeshCMD

### ISSUE THREAD

**Issue by zanaffar** [Ylianst/MeshCentral#354]
It would be very useful to be able to use MeshCMD to programmatically perform the same TLS certificate and private key enrollment steps that are currently only possible using the MeshCommander UI. Thank you.

**Comment by Ylianst** [comment]
Good idea. I am working on MeshCMD this week, this would be a good addition.

**Comment by Ylianst** [comment]
FYI. I did a bunch of updates to MeshCMD today. Added Intel AMT event log, static IP configuration and more. I did not get to enabling TLS yet.
